src/app/notebook/notebook-helpers.ts

```diff
@@ -0,0 +1,47 @@
+import { MimeBundle } from './notebook-types';
+
+// A list of the mime types we will display, ordered from "best" to "worst".
+export const supportedMimeTypes: string[] = ['image/png', 'image/webp', 'image/jpeg', 'image/gif', 'text/html', 'text/json', 'text/plain'];
```

Comment:
Makes me think if on the webservice side we want to tell people "you are uploading a notebook preview that includes mimetypes that won't render properly. Please refer to a doc or open a ticket if you require this functionality..."